### DISCUSSION TITLE
Is there a way to perform operation like LINQ `Any`?

### DISCUSSION THREAD

**Discussion by xperiandri** [Gremlinq/ExRam.Gremlinq#1382] [Q&A]
Is there a better way then below to find if a count is greater than 0 like `Any` in LINQ?
```
.Count()
.SingleOrDefaultAsync();
```

**Comment by danielcweber** [comment]
Would `ToAsyncEnumerable().Any()` work for you? It's also a good idea to use `Limit(1)` beforehand if it's just about determining whether count != 0.
